### PR TITLE
Dev continued UI improvements

### PR DESCRIPTION
**Pull request by heyitsjorge**
Register UI has been implemented.

Some comments from the previous CSS files have been added and cleaned up.

### CODE REVIEW

**Comment by curtisjbradley** [comment]
Looks good. Make sure to add a page header to the register page.

**Review by curtisjbradley — CHANGES_REQUESTED** [review]
Make sure the register page has a header.

**Comment by phillip-niakoulov** [comment]
Looks sick.

File: frontend/src/styles/pages/notfound.css
Comment (edsonmunoz) [inline frontend/src/styles/pages/notfound.css:1]: way to clean up the code

File: frontend/src/styles/pages/register.css
Comment (edsonmunoz) [inline frontend/src/styles/pages/register.css:1]: nice styles, looks good!!!

File: frontend/src/pages/Register.jsx [hunk 1]
```diff
@@ -38,15 +38,18 @@ function Register() {
...
-            <label htmlFor="username">Username:</label>
-            <input type="text" id="username" name="username" /> <br />
-            <label htmlFor="password">Password:</label>
-            <input type="password" id="password" name="password" /> <br />
-            <label htmlFor="confirm">Confirm Password:</label>
-            <input type="password" id="confirm" name="confirm" />
-            <p id={'error'}></p>
-            <input type="submit" onClick={submit_register} value="Submit" />
+        <div>
+            <h1>Register</h1>
```
Comment (curtisjbradley) [inline frontend/src/pages/Register.jsx:42]: Added Header

**Review by curtisjbradley — COMMENTED** [review]
Looks good.

**Pull request merged**